tools/export.ps1

```diff
@@ -0,0 +1,13 @@
+# Make an esp folder only if it does not yet exist, and move into it regardless
+$Filename="~/esp"
+if (-not(test-path $Filename)){mkdir $FileName}
+
+pushd ~/esp/esp-idf
+
+# Install tools
+./install.ps1
+
+# Export paths and variables
+./export.ps1
+
+popd
```

Comment:
I don't get what this is doing. What's the point of making an `~/esp/` folder and calling a script that may not exist in an empty folder?

Reply:
I think I must have stripped down the dev env setup script from last year to make this. Not sure why it does half the stuff it does, so I'll delete it.